packages/core/src/kernelCoreTemplate.js

```diff
@@ -211,16 +211,7 @@
 
       // create minimal selection if its a builtin and the whole path is not selected for
       if (!parentIsEntryModule && moduleData.type === 'builtin' && !parentPackagePolicy.builtin[moduleId]) {
-        const builtinPaths = (
```

Comment:
why was this moved to the toolkit file?

Reply:
It's more shareable that way and the logic of what it's doing is better encapsulated there.